GGSmtp/GGSmtp.ServerApp/MailServerLog4Net.cs

```diff
@@ -0,0 +1,78 @@
+﻿using log4net;
+using Simple.MailServer.Logging;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace GGSmtp.ServerApp
+{
+    public class MailServerLog4Net : IMailServerLogger
+    {
+        MailServerLogLevel mLogLevel;
+        ILog mLogger;
+
+        public MailServerLog4Net(MailServerLogLevel logLevel)
+        {
+            mLogLevel = logLevel;
+            mLogger = LogManager.GetLogger("MailServerLogger");
+        }
+
+        public MailServerLogLevel LogLevel
+        {
+            get
+            {
+                return mLogLevel;
+            }
+        }
+
+        public void Debug(string message)
+        {
+            if (mLogLevel >= MailServerLogLevel.Debug)
```

Comment:
refactor the if condition out to avoid code duplication throughout the class !fix